.gitpod/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM gitpod/workspace-full:latest
+
+USER gitpod
```

Comment:
I am wondering what is this for and do we really need this? I tried without this config and everything worked as expected.

Reply:
They are used to specify a base image but ig Gitpod already uses these settings by default, so they can be removed without issue.

Reply:
I think we can run as root (user) by default, since we are using Gitpod this would not be too risky

Reply:
Yes , right . I am removing both of these lines and will check again on gitpod just for surity.

Reply:
Wait, I meant for the `USER gitpod` line only

Reply:
I think they have some formatting issue on GitHub, so if I comment on a line plesae only pay attention to the last line. This is a known issue with GitHub.